job-dsls/jobs/pr_jobs.groovy

```diff
@@ -87,6 +87,14 @@ def final REPO_CONFIGS = [
         "jbpm-wb"                   : [
                 label: "kie-rhel7 && kie-mem16g"
         ],
+        "kie-docs"                   : [
+                label: "kie-rhel7 && kie-mem16g",
+                artifactsToArchive: DEFAULTS["artifactsToArchive"] + [
+                        "doc-content/drools-docs/target/generated-docs/html_single/**/*",
+                        "doc-content/jbpm-docs/target/generated-docs/html_single/**/*",
+                        "doc-content/optaplanner-wb-es-docs/target/generated-docs/html_single/**/*"
```

Comment:
we could limit the path to only one with starting ** or not? @jomarko is there a reason to have listed all 3 there?

`**/target/generated-docs/html_single/**/*"`

Reply:
@mareknovotny if all needed docs are in 
**/target/generated-docs/...
it makes sense to have only one line

Reply:
Yes sure, one string is fine, I was just not sure about wildcards so this was first thing I tried. Thanks for looking into it, both!